src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -105,11 +126,11 @@ export const SwapButton = ({
                   height={21}
                 />
                 <Box marginLeft={2}>
-                  <StyledPlataformText>{ROUTABLE_PLATFORM_STYLE[platformName].name}</StyledPlataformText>
+                  <StyledPlatformText>{ROUTABLE_PLATFORM_STYLE[platformName].name}</StyledPlatformText>
                 </Box>
               </Flex>
             )}
-            {priceImpactSeverity > PRICE_IMPACT_MEDIUM && ` ${t('button.anyway')}`}
+            <AnywayText>{priceImpactSeverity > PRICE_IMPACT_MEDIUM && t('button.anyway')}</AnywayText>
```

Comment:
Maybe we can use `UnifiedText`  here and add Box with margin left.

```suggestion
	<Box marginLeft={"0.25rem"}>
           <UnifiedText>{priceImpactSeverity > PRICE_IMPACT_MEDIUM && t('button.anyway')}</AnywayText>
         </Box>
```

Reply:
I would prefer to add one more styled component than add one more div to DOM, as the page is working slower. I know that it is insignificant in this case, but it makes a difference on a larger scale